src/lib/components/comparisonTable/index.stories.mdx

```diff
@@ -77,21 +77,32 @@ export const ComparisonTableWithData = () => {
           id: 4,
           key: 'imdb',
           label: 'IMDB rating',
+          render: (value) => (
+            <TableButton onClick={() => {}}>
+              {value}
+            </TableButton>
+          ),
         },
         {
           id: 3,
           key: 'rating',
-          label: <TableRowHeader label="Our Rating" onClickInfo={console.log} />,
+          label: <TableRowHeader label="Our rating of movies found online on IMDB lists" onClickInfo={() => {}} />,
           render: (value) => (
-            <TableButton className="jc-center" onClick={console.log}>
-              <TableRating type="star" rating={value} />
+            <TableButton onClick={() => {}}>
+              <TableRating type="stars" rating={value} />
```

Comment:
Apart from that everything looks good 👍 ✨

Reply:
Ah! Mdx + typescript isn't showing this and since it has the default value it all looked good from a testing perspective, just updated and will look into a way to make sure typescript, eslint and mdx all behave properly 🙇